store/storetest/webhook_store.go

```diff
@@ -201,8 +201,8 @@ func testWebhookStoreDeleteIncomingByChannel(t *testing.T, ss store.Store) {
 		t.Fatal("invalid returned webhook")
 	}
 
-	if r2 := <-ss.Webhook().PermanentDeleteIncomingByChannel(o1.ChannelId); r2.Err != nil {
-		t.Fatal(r2.Err)
+	if err := ss.Webhook().PermanentDeleteIncomingByChannel(o1.ChannelId); err != nil {
```

Comment:
Jenkins is complaining about this line because the `err` variable is shadowing the `err` variable at top level of the function, so just removing the `:` from the `:=` should do the trick :)

Reply:
@jespino fixed :)